utilities/post_migration.py

```
@@ -1395,6 +1431,16 @@ def check_vms(
                     )
                 except Exception as exp:
                     res[vm_name].append(f"check_static_ip_preservation - {str(exp)}")
+
+            # NIC name preservation check - for vSphere VMs with guest NIC names collected
+            if source_vm_data and source_provider.type == Provider.ProviderType.VSPHERE:
+                try:
+                    check_nic_name_preservation(
+                        source_vm_data=source_vm_data,
+                        destination_vm=destination_vm,
+                    )
+                except Exception as exp:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)